spec/workflow_service_spec.rb

```diff
@@ -189,10 +189,10 @@
       expect(Dor::WorkflowService.get_workflow_status('dor', 'druid:123', 'etdSubmitWF', 'registrar-approval')).to eq('completed')
     end
     it 'should throw an exception if it fails for any reason' do
-      expect{ Dor::WorkflowService.get_workflow_status('dor', 'druid:123', 'missingWF', 'registrar-approval') }.to raise_error Faraday::ResourceNotFound
+      expect{ Dor::WorkflowService.get_workflow_status('dor', 'druid:123', 'missingWF', 'registrar-approval') }.to raise_error Dor::WorkflowException
```

Comment:
Again, I'm not sure the specificity is useful downstream, and if it is, Ruby 2.1+ supports `Exception#cause`, allowing downstream rescuers to retrieve the underlying error.